src/pages/setpassword/SetPasswordForm/index.native.js

```diff
@@ -0,0 +1,3 @@
+import SetPasswordFormNarrow from './SetPasswordFormNarrow';
+
+export default SetPasswordFormNarrow;
```

Comment:
What if we have a large tablet like iPad Pro? Should we still show the "narrow" view?

Reply:
I think it would be great to make the form just one component, even if it means we have to make some design adjustments. For example, I'd be fine with the mobile view having everything left-aligned like the non-mobile view if that means we're using the same component.